presto-docs/src/main/sphinx/release/release-0.202.rst

```diff
@@ -0,0 +1,78 @@
+=============
+Release 0.202
+=============
+
+General Changes
+---------------
+
+* Fix correctness issue for queries involving aggregations over the result of an outer join
+  (`issue #10592 <https://github.com/prestodb/presto/issues/10592>`_).
+* Fix :func:`map` to fail on duplicate keys, instead of producing a corrupted map silently.
```

Comment:
"Fix ... to fail" sounds strange. Something like "Make ... fail on duplicate keys"